exporting/process_data.c

```diff
@@ -0,0 +1,270 @@
+// SPDX-License-Identifier: GPL-3.0-or-later
+
+#include "exporting_engine.h"
+
+/**
+ * Normalize chart and dimension names
+ *
+ * Substitute '_' for any special character except '.'.
+ *
+ * @param dst where to copy name to.
+ * @param src where to copy name from.
+ * @param max_len the maximum size of copied name.
+ * @return Returns the size of the copied name.
+ */
+size_t exporting_name_copy(char *dst, const char *src, size_t max_len)
+{
+    size_t n;
```

Comment:
This is the unique place of the file that you have a variable created outside the loop.

Reply:
Thanks, it's just a copy-paste of the old code. I'll fix it.

Reply:
`n` is used outside the loop. So the code won't be changed.